src/main/java/core/basesyntax/impl/ReadDataFromFileImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.service.ReadDataFromFile;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class ReadDataFromFileImpl implements ReadDataFromFile {
+    private static final String FILE_PATH = "src/main/java/core/basesyntax/input.csv";
```

Comment:
it is better to make the path an input parameter, because in this case this method will read only from one file, without the possibility to change it